### PR TITLE
add the IronCheckedElementBehaviour

### PR DESCRIPTION
**Pull request by notwaldorf**
:pager: @cdata

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
     "paper-styles": "PolymerLabs/paper-styles#^1.0.0",
     "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
+    "iron-checked-element-behavior": "PolymerElements/iron-checked-element-behavior#^1.0.0",
```
Comment (cdata) [inline bower.json:26]: :+1:

File: paper-radio-button.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
 <link rel="import" href="../paper-ripple/paper-ripple.html">
 <link rel="import" href="../paper-styles/default-theme.html">
 <link rel="import" href="../paper-behaviors/paper-inky-focus-behavior.html">
+<link rel="import" href="../iron-checked-element-behavior/iron-checked-element-behavior.html">
```
Comment (cdata) [inline paper-radio-button.html:14]: :+1:

File: paper-radio-button.html [hunk 3]
```diff
@@ -65,7 +66,8 @@
       is: 'paper-radio-button',
 
       behaviors: [
-        Polymer.PaperInkyFocusBehavior
+        Polymer.PaperInkyFocusBehavior,
+        Polymer.IronCheckedElementBehavior
```
Comment (cdata) [inline paper-radio-button.html:70]: :+1:

File: paper-radio-button.html [hunk 4]
```diff
@@ -87,38 +89,30 @@
...
-         * of the spacebar.
-         */
-        toggles: {
-          type: Boolean,
-          value: true,
-          reflectToAttribute: true
-        }
+         ariaActiveAttribute: {
+           value: 'aria-checked'
+         }
```
Comment (cdata) [inline paper-radio-button.html:94]: :+1:

File: paper-radio-button.html [hunk 5]
```diff
@@ -87,38 +89,30 @@
...
       },
 
       attached: function() {
-        var trimmedText = Polymer.dom(this).textContent.trim();
-        if (trimmedText === '') {
-          this.$.radioLabel.hidden = true;
-        }
+        this._isReady = true;
```
Comment (cdata) [inline paper-radio-button.html:98]: Is this still needed?

File: paper-radio-button.html [hunk 6]
```diff
@@ -87,38 +89,30 @@
...
+      /**
+       * Update the checkbox aria-label. This is a temporary workaround not
+       * being able to observe changes in <content>
+       * (see: https://github.com/Polymer/polymer/issues/1773)
+       *
+       * Call this if you manually change the contents of the checkbox
+       * and want the aria-label to match the new contents.
+       */
+      updateAriaLabel: function() {
+        this.setAttribute('aria-label', Polymer.dom(this).textContent.trim());
```
Comment (cdata) [inline paper-radio-button.html:115]: :+1:

File: paper-radio-button.html (hunk 5 above)
Comment (notwaldorf) [inline paper-radio-button.html:98]: Yes 😢

`_buttonStateChanged` (see L122) is still called before `attached`, and stomps over the user-set `checked` attribute. :sob:

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**